scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -1907,6 +1907,30 @@ class FormatOps(
     }
   }
 
+  // Optional braces in templates after `:|with`
+  object OptionalBracesTmpl {
```

Comment:
```suggestion
  object OptionalBracesTemplate {
```
?

Reply:
:) will do. i think it was overflowing the line somewhere.

Reply:
done, uploaded